src/sources/docker_logs.rs

```diff
@@ -51,6 +51,7 @@ lazy_static! {
 #[derive(Deserialize, Serialize, Debug, Clone)]
 #[serde(deny_unknown_fields, default)]
 pub struct DockerLogsConfig {
+    exclude_containers: Option<Vec<String>>,
```

Comment:
Nit: I might duplicate the comment for `include_containers` that this is a prefix match.